src/Resources/Resources/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated `Get-AzResourceGroupDeploymentOperation`to use the SDK.
+* Remove ApiVersion parameter
```

Comment:
I think it's a worthy effort to make the change longs a bit verbose to make things clear. We can add something like "Remove ApiVersion parameter from Deployment cmdlets (if the cmdlet list is not too long, maybe add them too?)

Reply:
There are quite a few cmdlets (mentioned in the design review), so I don't think adding them to the changelog is practical. But I'll make it a little more verbose.